### DISCUSSION TITLE
Volume condition for face elements (pdim = 3, cdim = 2)

### DISCUSSION THREAD

**Discussion by amalsahai** [CGNS/CGNS#792] [CGNStalk]
I'm outputting a surface mesh for a closed 3D object comprising of TRI_3 elements. I would like to be able to identify patches of triangles which require different treatments once the mesh has been read in. What would be the most appropriate way to flag this in a CGNS file? I'm looking for something akin to volume conditions but for cdim = 2, pdim = 3. Is that possible?

**Comment by MicK7** [comment]
It should be possible. CGNS ZoneSubRegion seems to be what you are looking for.
